### PR TITLE
third_party: update erigon-interfaces to commit cd6dd2549

### PR DESCRIPTION
**Pull request by lupin012**

### CODE REVIEW

File: silkworm/rpc/ethbackend/backend.hpp [hunk 1]
```diff
@@ -45,6 +45,7 @@ class BackEnd {
     virtual Task<BlockNum> get_block_number_from_txn_hash(const HashAsSpan& hash) = 0;
     virtual Task<BlockNum> get_block_number_from_hash(const HashAsSpan& hash) = 0;
     virtual Task<evmc::bytes32> get_block_hash_from_block_number(uint64_t number) = 0;
+    virtual Task<std::string> canonical_body_for_storage(uint64_t number) = 0;
```
Comment (canepat) [inline silkworm/rpc/ethbackend/backend.hpp:48]: 1. `uint64_t number` -> `BlockNum block_number`
2. `Task<std::string>` -> `Task<Bytes>`

File: silkworm/rpc/ethbackend/remote_backend.cpp [hunk 2]
```diff
@@ -202,6 +202,17 @@ Task<evmc::bytes32> RemoteBackEnd::get_block_hash_from_block_number(uint64_t num
     co_return hash;
 }
 
+Task<std::string> RemoteBackEnd::canonical_body_for_storage(uint64_t number) {
+    const auto start_time = clock_time::now();
+    UnaryRpc<&::remote::ETHBACKEND::StubInterface::AsyncCanonicalBodyForStorage> canonical_body_for_storage_rpc{*stub_, grpc_context_};
+    ::remote::CanonicalBodyForStorageRequest request;
+    request.set_blocknumber(number);
+    const auto reply = co_await canonical_body_for_storage_rpc.finish_on(executor_, request);
+    SILK_TRACE << "RemoteBackEnd::canonical_body_for_storage bn="
```
Comment (canepat) [inline silkworm/rpc/ethbackend/remote_backend.cpp:211]: Add missing `<< block_number` at the end of this line

File: silkworm/rpc/ethbackend/remote_backend.cpp [hunk 3]
```diff
@@ -202,6 +202,17 @@ Task<evmc::bytes32> RemoteBackEnd::get_block_hash_from_block_number(uint64_t num
...
+               << " t=" << clock_time::since(start_time);
+    co_return reply.body();
```
Comment (canepat) [inline silkworm/rpc/ethbackend/remote_backend.cpp:213]: After changing return type to `Task<Bytes>`, you should also check which data the `body` string is carrying on and handle the decoding here (as opposed to putting such burden on each call site).

**Review by canepat — CHANGES_REQUESTED** [review]

File: silkworm/rpc/ethbackend/backend.hpp (hunk 1 above)
Comment (lupin012) [reply silkworm/rpc/ethbackend/backend.hpp]: ok

File: silkworm/rpc/ethbackend/remote_backend.cpp (hunk 2 above)
Comment (lupin012) [reply silkworm/rpc/ethbackend/remote_backend.cpp]: ok

File: silkworm/rpc/ethbackend/remote_backend.cpp (hunk 3 above)
Comment (lupin012) [reply silkworm/rpc/ethbackend/remote_backend.cpp]: I don't understand if the return types remains Bytes or it is necessary calls decode_stored_block_body() and so the returns types becomes Task<BlockBodyForStorage>

Comment (canepat) [reply silkworm/rpc/ethbackend/remote_backend.cpp]: You're right, it's better to change the return type to `BlockBodyForStorage`

**Review by canepat — APPROVED** [review]

**Pull request merged**